src/main/java/org/jabref/gui/fieldeditors/PersonsEditor.java

```diff
@@ -23,12 +23,10 @@ public PersonsEditor(final Field field,
                          final SuggestionProvider<?> suggestionProvider,
                          final JabRefPreferences preferences,
                          final FieldCheckers fieldCheckers,
-                         final boolean isSingleLine) {
+                         final boolean isMultiLine) {
         this.viewModel = new PersonsEditorViewModel(field, suggestionProvider, preferences.getAutoCompletePreferences(), fieldCheckers);
 
-        textInput = isSingleLine
-                ? new EditorTextField()
-                : new EditorTextArea();
+        textInput = isMultiLine ? new EditorTextArea() : new EditorTextField();
```

Comment:
Not that the true/false branches were just be wrapped. Another indicator for an enum!